src/index.ts

```diff
@@ -106,7 +120,7 @@ export class UnleashClient extends TinyEmitter {
         }
     }
 
-    public updateContext(context: IContext) {
+    public updateContext(context: IMutableContext) {
```

Comment:
If this results in a breaking change, we could add another method which does this but update the old method to do the same thing with less strict types.

Reply:
I think it is fine, but maybe we should add a console warning if the context object contains either `appName` og `environment`? The typescript definition only helps if the developer uses the type definition.

Reply:
Something like this @ivarconr? ed1ab8b But would the user even get this far? I would hope even if a user is consuming the package with js, ts would still catch that context is not the right shape and throw an error.

Reply:
yes. Looking at the code, this is not changed from how it was before, but still nice to help the developers.